CHANGELOG.md

```diff
@@ -1,3 +1,107 @@
+## 6.12.0
+
+### System Improvements
+
+- (Michael) Support delta inputs for health and shield hp in the character sheet
+- (Supe) Convert roll inspector to application v2
+
+### Bugfixes
+
+- (Dire Weasel) Fix missing localized string for NPC Shield Max HP
+- (Supe) Avoid persisting automated weapon property runes when closing the item sheet
+- (Supe) Fix dragging and dropping persistent damage
+- (Supe) Fix invisible tokens absorbing drag/drop events on the canvas
+- (Supe) Fix penalties that don't double on crit such as cringe when rolling critical damage
+- (Supe) Fix trick magic item spell DC when using trained or expert skills
+- (Supe) Show description item alterations for effect panel tooltips and expandable self effect actions
+- (Supe) Support multiple token marks instead of overriding them
+- (Vauxs) Exclude eidolons from ABC Picker
+
+### Data Updates
+
+- (Ambrose) Add automation to Hardshell Surki and Conductive Sphere
+- (Ambrose) Add missing price for Aurochs Jerky
+- (Ambrose) Add options introduced in the 2025 April Fool's Paizo Blog
+- (Ambrose) Condense equipment effects and add icons to equipment missing them
+- (Ambrose) Correct minor issues in Paragon Medicine, Wellspring Magic, Unfolding Tree House
+- (Ambrose) Update Archetype journal entry formatting, GM Screen content
+- (Ambrose) Correct Inflammation Flask's damage type and update effect automation
+- (Ambrose) Correct issues in Neshen and Sivanah deity items
+- (Ambrose) Extend Unstable note to actions that have the `unstable` trait and add notes RE to Inventor dedication
+- (Ambrose) Fix automation of Magnetic Shot
+- (Ambrose) Fix issues and improve automation in several NPCS
+- (Ambrose) Link various `Bloodstone of Arzani` effects to their corresponding items
+- (Ambrose) Move `Triumph of the Tusk` backgrounds to their folder under Adventure Paths
+- (Ambrose) Move Beginner Box Pregens backstory text to the `Backstory` section
+- (Ambrose) Remove errant traits from Elemental Counter and Grasp of Droskar
+- (Ambrose) Remove Cooperative Soul's prerequisite
+- (Ambrose) Remove invalidated feats and spells
+- (Ambrose) Update Anima Invocation spell effect to remaster rules
+- (Daomephsta) Fix a few enricher syntax typos in creatures and hazards
+- (Dire Weasel) Add automation for Dragon's Flight,  Magus's Analysis
+- (Dire Weasel) Add bonus against fear to Effect: Swig
+- (Dire Weasel) Add custom IWR to Phantasmal Minion and Armag Twice-Born
+- (Dire Weasel) Add effect for Animalistic Brutality, Blessed Coins, Glimpse Vulnerability, Aurochs Jerky, and Cindergrass Cloak
+- (Dire Weasel) Add effect link for Sinful Bite
+- (Dire Weasel) Add initiative Note to Cloud of Allergens
+- (Dire Weasel) Add predicate to 's Belief Fortification
```

Comment:
```suggestion
- (Dire Weasel) Add predicate to Garrholdion's Belief Fortification
```

Reply:
Actually, it looks like there is already an automation improvement entry for that creature. I'm thinking that perhaps it wasn't mergeed properly because of this typo. I just removed it.